tests/treeTest.py

```diff
@@ -0,0 +1,19 @@
+from coalahtml.tree import Tree
+import unittest
+
+
+class TreeTest(unittest.TestCase):
+
+    def setUp(self):
+        self.FL = [
+                    "/home/tushar/coala-html/tests",
+                    "/home/tushar/coala-html/coalahtml",
+                    "/home/tushar/coala-html/tests/specs/directives/",
+                    "/home/tushar/coala-html/setup.py",
+                    "/home/tushar/coala-html/tests/TestUtilities.py"
+                  ]
+
+    def test_graph(self):
+        tr = Tree(self.FL)
+        testChild = tr.root.child['tests'].child.get('specs', None)
+        self.assertTrue(testChild)
```

Comment:
OK, but is that empty node at the intentional? IMO you should have tests for this anyway.